interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -1,33 +1,57 @@
 <!--
   TODO each sample should be built using a reusable Angular directive
 -->
-<h1>Samples <small>(drag to reorder)</small></h1>
+<h1>Samples
+  <small>(drag <i class="fa fa-bars" aria-hidden="true"></i> to reorder,
+    click <i class="fa fa-times" aria-hidden="true"></i> to delete,
+    choose <input type="radio" disabled> to set group)</small>
+</h1>
 <status-bar msg-text="analysis.status"></status-bar>
 <!-- TODO need a tool to select and order annotation columns -->
 <table class="table table-hover">
   <thead>
     <tr>
-      <td>&nbsp;</td>
-      <td>id</td>
-      <td>ML Data Source</td>
-      <td>Sample Name</td>
-      <td ng-repeat="at in analysis.exampleCols">{{
+      <th> </th>
+      <th>
+        <span class="row-control">
+          <i class="fa fa-bars" aria-hidden="true"></i>
+        </span>
+        ML Data Source
+      </th>
+      <th>id</th>
+      <th>Sample Name</th>
+      <th ng-repeat="at in analysis.exampleCols">{{
         at.typename
-      }}</td>
+      }}</th>
     </tr>
   </thead>
   <tbody ng-model="sb.heatmapData.samples" as-sortable="sortableOptions">
-    <tr ng-repeat="id in sb.heatmapData.samples" as-sortable-item>
-      <td>
-        <button type="button" class="btn close"
-        ng-click="sb.removeSample(id)" aria-label="Delete">
+    <tr ng-repeat="id in sb.heatmapData.samples"
+        ng-mouseover="showControls = true" ng-mouseleave="showControls = false"
+        ng-class="'sample-' + sb.sampleGroups[id]" as-sortable-item>
+      <td class="row-control">
+        <input type="radio" ng-model="sb.sampleGroups[id]"
```

Comment:
On my web browser (Google chrome), these three radio buttons are not very easy to find because they are gray. How easy is it to make them more obvious? (If it needs a lot of tweaking, you can do it sometime later.)